gamemeth.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include "game.h"
+#include <string>
+
+//a Game oszt�ly met�dusai kifejtve, le�r�s az oszt�lyban
+
+Game::Game(Characters A, Characters B) {
+	this->A = A;
+	this->B = B;
+}
+
+void Game::Fight() {
+	//Elind�l a j�t�k
+	std::cout << "Game start" << "\n" << std::endl;

```

Comment:
Lejjebb is láttam: `<< "\n" << std::endl;` Ezt miért így? Miért nem `<< "\n\n"`?

Reply:
OOP-n preferált volt, legalábbis emlékszem hogy mindenhol ez írtuk. Persze a \n is flush ol ez inkább amolyan biztos ami biztos. Sokat gondolkodok rajta mert szokták írni hogy ronthat a futási teljesítményen

Reply:
Értem. Ha fontos, hogy azonnal flush-öljön, akkor valóban endl kell, de itt nincs rá szükség, mint ahogy a legtöbb esetben sincs. Sokat nem érdemes ezen gondolkodni, általában nem ez a fő teljesítményt befolyásoló tényező. Szóval nyugodtan lehet `std::cout << "Game start\n\n";` és akkor rövidebb a kód.